synedrion/src/www02.rs

```diff
@@ -1,5 +1,8 @@
 mod entities;
 pub(crate) mod key_resharing;
 
-pub use entities::{DeriveChildKey, ThresholdKeyShare};
+pub use bip32::DeriveChildKey;
+pub use entities::ThresholdKeyShare;
 pub use key_resharing::{KeyResharing, KeyResharingProtocol, NewHolder, OldHolder};
+
+mod bip32;
```

Comment:
Not sure it should be a part of `www02` - look more suitable for `curve` submodule.

Reply:
Agreed. Moving.